File: Iniciante/URI 1009.cs
using System; 
 
class URI {
 
    static void Main(string[] args) { 
        string nome = Console.ReadLine();
        double salarioFixo = double.Parse(Console.ReadLine());
        double vendas = double.Parse(Console.ReadLine());
        double total = ((vendas * 15) / 100) + salarioFixo;
        Console.WriteLine("TOTAL = R$ {0:F2}", total);
    }
    
}

File: Iniciante/URI 1009.dart
import 'dart:io';

void main() {
  String nome = stdin.readLineSync();
  double salarioFixo = double.parse(stdin.readLineSync());
  double vendas = double.parse(stdin.readLineSync());
  double total = ((vendas * 15) / 100) + salarioFixo;
  print("TOTAL = R\$ ${total.toStringAsFixed(2)}");
}
